openedx/core/djangoapps/schedules/management/commands/__init__.py

```diff
@@ -0,0 +1,128 @@
+import datetime
+import logging
+
+import pytz
+from django.contrib.sites.models import Site
+from django.core.management.base import BaseCommand
+from edx_ace.recipient_resolver import RecipientResolver
+
+from openedx.core.djangoapps.schedules.models import ScheduleConfig
+from openedx.core.djangoapps.schedules.tasks import DEFAULT_NUM_BINS
+from openedx.core.djangoapps.site_configuration.models import SiteConfiguration
+
+
+LOG = logging.getLogger(__name__)
+
+
+class PrefixedDebugLoggerMixin(object):
```

Comment:
Nit: consider using a [LoggerAdapter](https://docs.python.org/2/library/logging.html#logging.LoggerAdapter) instead

Reply:
I think in the interest of getting this merged quickly we should add a `#TODO` here and hope we get around to doing that at some point?